nrf_security/Kconfig

```diff
@@ -1322,6 +1322,11 @@ endif # MBEDTLS_RSA_C
 
 menu "SHA   - Secure Hash Algorithm"
 
+config MBEDTLS_MD5_C
+	bool
+	help
+		MD5 hash functionality.
```

Comment:
I think the indentation is off. It should be tab + 2 spaces

Reply:
Modified accordingly and force-pushed.